sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -1241,12 +1401,6 @@ static char sdw_card_long_name[] = "sof-sdw-rt711-stereo-rt1308-rt715";
 #endif
 static struct snd_soc_card card_rt700_rt1308_rt715 = {
 	.name = "sdw-rt711-1308-715",
-	.controls = controls,
```

Comment:
@plbossart this is so confusing. card_rt700_rt1308_rt715 has a name "sdw-rt711-1308-715"? Is it a typo?

Reply:
yes it's a typo but it's removed in the last patch